nntrainer/tensor/var_grad.cpp

```diff
@@ -26,6 +26,10 @@ Var_Grad::Var_Grad(const TensorDim &dim, const Tensor::Initializer init,
   name(name) {
   var = std::make_shared<Tensor>(dim, alloc_now, init);
   if (need_gradient)
+    /**
+     * @todo gradient initializer should be none, and then they should be set
```

Comment:
I guess this comment is related to the question?

Reply:
yes, this is the corresponding to do for the answer above.